documentation/operators/src/sandbox.md

```diff
@@ -0,0 +1,50 @@
+# Sandbox Testnet
+
+Nym node operators can run their nodes on Nym Sandbox testnet environment. Whether it's testing new configuration, hot features from Nym developers or just trying to setup a node for the first time, this environment is for you. Below are steps to [setup your environment](#sandbox-environment-setup) and an introduction to [Sandbox token faucet](#sandbox-token-faucet).
+
+```admonish warning title=""
+This page is for Nym node operators. If you want to run NymVPN CLI over Sandbox testnet, visit our [developers portal](https://nymtech.net/developers/nymvpn/cli.html#testnet-environment).
+```
+
+## Sanbox Environment Setup
+
+> Any syntax in `<>` brackets is a user's unique variable. Exchange with a corresponding name without the `<>` brackets.
+
+To run Nym binaries in Sandbox Testnet you need to get the `.env` configuration file and point your binary to it. Follow the steps below:
+
+
+1. Create Sandbox environment config file by saving [this](https://raw.githubusercontent.com/nymtech/nym/develop/envs/sandbox.env) as `sandbox.env` in the same directory as your binaries:
+```sh
+curl -o sandbox.env -L https://raw.githubusercontent.com/nymtech/nym/develop/envs/sandbox.env
+```
+
+2. Run your `nym-node` with all the commands as always with an additional flag `--config-file` with a path to `sanbox.env` file. For example:
+```sh
+# this example is for mixnode mode
+./nym-node run --mode mixnode --config-file <PATH/TO/sandbox.env>
```

Comment:
I think this `--config-file` is the path to the nym-node configuration. 

It should instead be
`./nym-node --config-env-file sandbox.env  run --mode mixnode` 
or simply 
`./nym-node -c sandbox.env  run --mode mixnode`

Reply:
wtf, of course you are right!